### PR TITLE
Use Ember.assign instead of Ember.merge if possible

### PR DESCRIPTION
**Pull request by offirgolan**
Resolves offirgolan/ember-light-table#121

### CODE REVIEW

**Comment by poteto** [comment]
Thanks, re-running Travis

**Comment by offirgolan** [comment]
@poteto no problem! Travis passed 😺. Let me know when a patch has been released

**Comment by poteto** [comment]
Released in 2.0.8

**Comment by offirgolan** [comment]
@poteto perfect! Thank you

**Comment by danorton2** [comment]
Did you miss one?
https://github.com/DockYard/ember-in-viewport/blob/master/app/initializers/viewport-config.js#L29

**Comment by poteto** [comment]
doh! want to open a PR to fix it? :)

File: addon/mixins/in-viewport.js [hunk 1]
```diff
@@ -63,7 +65,7 @@ export default Mixin.create({
     const owner = getOwner(this);
 
     if (owner) {
-      return merge(defaultOptions, owner.lookup('config:in-viewport'));
+      return assign(defaultOptions, owner.lookup('config:in-viewport'));
```
Comment (danorton2) [inline addon/mixins/in-viewport.js]: This looks like it clobbers defaultOptions, no?

Comment (danorton2) [inline addon/mixins/in-viewport.js]: Should be:

    return assign({}, defaultOptions, owner.lookup('config:in-viewport'));

**Pull request merged**